ImageLounge/src/DkGui/DkWidgets.h

```diff
@@ -436,6 +436,7 @@ public slots:
     QTreeView *mFileTree = nullptr;
     DkFileSystemModel *mFileModel = nullptr;
     DkSortFileProxyModel *mSortModel = nullptr;
+    QString *mCurrentFilePath = nullptr;

```

Comment:
```suggestion
    QString mCurrentFilePath {};

```

QString handles the dynamically allocated memory internally, so there is no need to use a pointer here.